test/nuxeo-page-provider.html

```diff
@@ -97,6 +103,56 @@
 
         });
 
+        suite('when page provider returns an error response', () => {
+
+          setup(() => {
+            server.respondWith('GET', '/json/cmis', [
+              200,
+              responseHeaders.json,
+              '{}',
+            ]);
+            server.respondWith(
+              'POST',
+              '/api/v1/automation/login',
+              [
+                200,
+                responseHeaders.json,
+                '{"entity-type":"login","username":"Administrator"}',
+              ],
+            );
+            server.respondWith(
+              'GET',
+              '/api/v1/user/Administrator',
+              [
+                200,
+                responseHeaders.json,
+                '{"entity-type":"login","username":"Administrator"}',
+              ],
+            );
+          });
+
+          test('it should throw an error', () => {
+            server.respondWith(
+              'GET',
+              '/api/v1/search/pp/test_provider/execute?currentPageIndex=0&pageSize=40',
+              [
+                500,
+                responseHeaders.json,
+                '{"entity-type":"exception", "message":"Internal Server Error", "status": 500}',
+              ],
+            );
+            const pp = fixture('nx-pp');
+            return pp.fetch().then(() => {
+              throw new Error('Expected to have an error response!');
+            }).catch((error) => {
+              expect(error).to.exist;
```

Comment:
If this is where the lint fails:
```suggestion
              expect(error).to.exist; // eslint-disable-line no-unused-expressions
```